### ISSUE TITLE
Package Caster for Linux as a Flatpak

### ISSUE THREAD

**Issue by Kentoseth** [dictation-toolbox/Caster#942]
**Is your feature request related to a problem? Please describe.**

Caster is confusing to install. It doesn't exist in many package managers and not being in the AUR means we are probably at the bleeding edge of software.

Yet Caster seems to be the only FOSS alternative to Talon. 

Packaging Caster into a Flatpak will solve Linux distribution issues. The folks at https://github.com/mkiol/dsnote did this and it is super easy to get voice dictation on Linux now. Total OS control would be even more awesome.

**Describe the solution you'd like**

Caster Flatpak packaged with an Open Source engine (Kaldi?) that works out of the box.

**Describe alternatives you've considered**

AppImage or Snap? Flatpak seems to be the most viable here.

**Additional context**

The Linux community is not really aware of Caster but RSI is a big problem. Packaging it for easy distribution will get it into more hands/usage and increase adoption, especially for people who need this software.

**Comment by LexiconCode** [comment]
I agree easy install is definitely an area for improvement! Although not super familiar with packaging but I can take a look.  I wouldn't mind some help in this issue.

**Comment by Kentoseth** [comment]
I will try to create a Flatpak, but I recommend that you have control of the FlatHub account to distribute the Flatpak.
